<ui version="4.0" >
 <class>TextureEditorClass</class>
 <widget class="QWidget" name="TextureEditorClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>954</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TextureEditor</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Texture</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxAction" >
     <property name="title" >
      <string> Action selection </string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="moveButton" >
        <property name="text" >
         <string>View</string>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectButton" >
        <property name="text" >
         <string>Select</string>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editButton" >
        <property name="text" >
         <string>Edit</string>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="optimizeButton" >
        <property name="text" >
         <string>Optimize</string>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="optionButton" >
        <property name="text" >
         <string>Option</string>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxUVMapEdit" >
     <property name="title" >
      <string> UV Map editing </string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" >
       <widget class="QGroupBox" name="groupBox_4" >
        <property name="title" >
         <string> Remap </string>
        </property>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="clampButton" >
           <property name="text" >
            <string>Clamp</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="modulusButton" >
           <property name="text" >
            <string>Modulus</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QGroupBox" name="groupBox_5" >
        <property name="title" >
         <string> Rotate </string>
        </property>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QSpinBox" name="spinBoxAngle" >
           <property name="maximum" >
            <number>360</number>
           </property>
           <property name="minimum" >
            <number>-360</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>degree</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="rotateButton" >
           <property name="text" >
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QGroupBox" name="groupBox" >
        <property name="title" >
         <string> Mode </string>
        </property>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButtonMove" >
           <property name="text" >
            <string>Move component</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonSelect" >
           <property name="text" >
            <string>Select origin</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QGroupBox" name="groupBox_6" >
        <property name="title" >
         <string> Scale </string>
        </property>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QSpinBox" name="spinBoxScale" >
           <property name="maximum" >
            <number>300</number>
           </property>
           <property name="minimum" >
            <number>10</number>
           </property>
           <property name="singleStep" >
            <number>5</number>
           </property>
           <property name="value" >
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>%</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="scaleButton" >
           <property name="text" >
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxSelect" >
     <property name="title" >
      <string> Select mode  </string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButtonArea" >
        <property name="text" >
         <string>Area</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonConnected" >
        <property name="text" >
         <string>Connected component</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonClear" >
        <property name="text" >
         <string>  Clear selection  </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxOptimize" >
     <property name="title" >
      <string> Optimize </string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="6" >
       <widget class="QPushButton" name="smoothButton" >
        <property name="text" >
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QSpinBox" name="spinBoxAlfa" />
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Alfa:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2" >
       <widget class="QLabel" name="label_4" >
        <property name="font" >
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text" >
         <string>Smooth texture cootdinates</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4" colspan="4" >
       <widget class="QProgressBar" name="progressBar" >
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="value" >
         <number>0</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3" >
       <widget class="QLabel" name="StatusLabel" >
        <property name="font" >
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text" >
         <string>Loading...</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy" >
         <cstring></cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxOption" >
     <property name="title" >
      <string> Options </string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="2" >
       <widget class="QSpinBox" name="spinBox_2" />
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Color:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QSpinBox" name="spinBox_3" />
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="spinBox" />
      </item>
      <item row="0" column="4" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
